cfgov/hmda/tests/test_outage_banner.py

```diff
@@ -0,0 +1,20 @@
+from django.template import Context, Template
+from django.test import RequestFactory, SimpleTestCase
+
+from flags.sources import get_flags
+
+
+class TestHmdaOutageBannerTag(SimpleTestCase):
+
+    def render(self, path):
+        request = RequestFactory().get(path)
+        request.flag_conditions = get_flags()
+        template = Template(
+            '{% load hmda_banners %}'
+            '{% hmda_outage_banner request %}'
+        )
+        return template.render(Context({'request': request}))
+
+    def test_banner_renders(self):
```

Comment:
Would you mind adding an additional test to show that the banner doesn't show up when the flag conditions aren't met?

Reply:
I have a confession: this test is a farce that doesn't actually use the URL to test the flag's logic. Please don't tell @chosak. I will rewrite it to make it more useful before @chosak notices. I really hope @chosak doesn't see this.

Reply:
I pushed an update to add tests showing that the banner conforms to the flag conditions.